package.json

```diff
@@ -26,6 +26,7 @@
     "@types/eventemitter3": "1.2.0",
     "addressbar": "1.0.3",
     "aphrodite": "1.2.0",
+    "baobab": "^2.4.3",
```

Comment:
Should this be a dependency? If someone want´s immutable they need to install it?